glossary.yml

```
@@ -1870,6 +1894,18 @@
       The use of computers to search for patterns in large datasets. The term [data
       science](#data_science) is now more commonly used.
 
+- slug: data_munging
+  ref:
+    - data_wrangling
+  en:
+    term: "data wrangling"
+    def: >
+      A colloquial name for small-scale [data engineering](#data_engineering).
```

I disagree with this. I don't believe this term is only used in cases where the data are small; I don't believe it is only used colloquially; and the wording sounds a bit like 'only data engineers working with very large data do data engineering, everyone else is just wrangling data'.

I'm going to cite O'Reilly here, in their [definition of data engineering](https://www.oreilly.com/content/data-engineering-a-quick-and-simple-definition/#:~:text=Data%20engineers%20make%20sure%20the,use%20cases%20may%20present%20themselves.&text=Data%20wrangling%20is%20about%20taking,turning%20it%20into%20something%20useful.) they say this:

> Data engineers wrangle data into a state that can then have queries run against it by data scientists.

I would suggest the definition be something like this:
```suggestion
      Another name for small-scale [data engineering](#data_engineering).
```
or:
```suggestion
      Some of the work done by [data engineers](#data_engineer).
```

---

I actually agree with you. I wasn't so sure about this definition in English (my own work with genomics data is far from "small-scale"). To clarify, it was already defined in English here:

https://github.com/carpentries/glosario/blob/master/glossary.yml#L1928-L1932
https://github.com/carpentries/glosario/blob/89cfe50c175a484cce072354dc18c1530d1aabd8/glossary.yml#L1928-L1932

What I've done is copied it from "data_wrangling" to "data_munging" and given a Japanese translation. As you rightly point out in PR #298 I confused the "ref" section for synonyms and I'm still not sure how to handle them. Does `glosario` project have any guidance on synonyms already?

I'm open to changing the original definition.  It doesn't clearly describe what it involves to someone unfamiliar with "data engineering" either. It seems to have been contributed originally here: https://github.com/carpentries/glosario/pull/65

---

However I think stating is colloquial is important. https://github.com/carpentries/glosario/pull/303#issuecomment-816723004